gio/ginetaddress.c

```diff
@@ -93,9 +101,9 @@ enum
 
 static void
 g_inet_address_set_property (GObject      *object,
-			     guint         prop_id,
-			     const GValue *value,
-			     GParamSpec   *pspec)
+                             guint         prop_id,
+                             const GValue *value,
+                             GParamSpec   *pspec)
```

Comment:
Let's avoid making unnecessary changes, as it will make future patch rebase operations heavier.